packages/web/src/components/withdraw-usdc-modal/WithdrawUSDCModal.tsx

```diff
@@ -113,12 +151,13 @@ export const WithdrawUSDCModal = () => {
       dispatch(
         beginWithdrawUSDC({
           amount,
+          currentBalance: balanceNumberCents,
```

Comment:
divide by 100?

Reply:
oops yes. I don't love all of the places I had to do this.